cax/main.py

```diff
@@ -288,10 +292,47 @@ def massive():
         docs = list(collection.find(query,
                                     sort=sort_key,
                                     projection=['start', 'number','name',
-                                                'detector', '_id']))
+                                                'detector', '_id', 'processor']))
 
         for doc in docs:
+            
+            #Run certains actions only on local file.
+            #If only_local is chosen, no jobs will submitted to batch queues
+            if only_local == True:
+                #ask if data field processor/correction_versions exists
+                #if not start to apply corrections
+                if 'processor' in doc:
+                    if 'correction_versions' not in doc['processor']:
```

Comment:
can you add another flag that toggles this requirement? (e.g. in case we want to run in local mode regardless)